### PR TITLE
[MDB IGNORE] Grayscale sofas

### PR DESCRIPTION
**Pull request by dragomagol**
## About The Pull Request
Ports some sofa sprites from over at https://github.com/BeeStation/BeeStation-Hornet/pull/7547 and makes them grayscale. Also random tool colours now use defines.

image

image

## Why It's Good For The Game
Gives us the opportunity to make different coloured couches more easily. 
## Changelog

:cl: Tattle, PestoVerde322
imageadd: couches are now grayscale
code: random tool colours now use defines
/:cl:

### CODE REVIEW

**Comment by dragomagol** [comment]
I do intend to make them recolourable in-game but that'll be probably in another PR

File: _maps/map_files/KiloStation/KiloStation.dmm [hunk 1]
```diff
@@ -58765,7 +58765,7 @@
 /turf/open/floor/plating,
 /area/station/engineering/atmos/storage/gas)
 "qPv" = (
-/obj/structure/chair/sofa/left{
+/obj/structure/chair/sofa/left/brown{
 	color = "#c45c57";
```
Comment (Fikou) [inline _maps/map_files/KiloStation/KiloStation.dmm:58769]: "brown"

**Review by Twaticus — APPROVED** [review]

**Comment by Mothblocks** [comment]
aw i liked the grime :(

File: code/__DEFINES/sofas.dm [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/// Create colored subtypes for sofas
+#define COLORED_SOFA(path, color_name, sofa_color) \
```
Comment (Mothblocks) [inline code/__DEFINES/sofas.dm:2]: Just put this in the sofa file itself, it shouldn't be used outside of that

**Review by Mothblocks — APPROVED** [review]
just my opinion but can u bring back the grime

File: code/game/objects/items/tools/wirecutters.dm [hunk 3]
```diff
@@ -33,19 +33,18 @@
...
+		COLOR_TOOL_GREEN,
+		COLOR_TOOL_CYAN,
+		COLOR_TOOL_YELLOW,
 	)
 
 /obj/item/wirecutters/Initialize(mapload)
 	if(random_color)
-		var/our_color = pick(wirecutter_colors)
-		set_greyscale(colors=list(wirecutter_colors[our_color]))
+		set_greyscale(colors = list(pick(wirecutter_colors)))
```
Comment (JohnFulpWillard) [inline code/game/objects/items/tools/wirecutters.dm:47]: why was it made like this? player-color editing? vending machines that let you pick their color?

I just want to be sure nothing will break by this.

Comment (dragomagol) [reply code/game/objects/items/tools/wirecutters.dm:47]: If there has ever been a way to choose which colour you can make your screwdriver/wirecutters, it's not known to me.

**Review by Maurukas — APPROVED** [review]
Maps and path update look good.

**Comment by optimumtact** [comment]
can the three people who approved this PR who had merge priveleges but didnt' merge it explain why

**Comment by Mothblocks** [comment]
i didnt notice twaticus already approved it

**Pull request merged**